src/docs/product/sentry-basics/search/saved-searches.mdx

```diff
@@ -2,72 +2,55 @@
 title: Saved Searches
 sidebar_order: 20
 keywords: ["saved search", "premade searches"]
-description: "Learn more about recommended, pinned, and saved searches."
+description: "Learn more about default, recommended, and saved searches."
 ---
 
-In the "Saved Searches" tab of **Issues**, you can access recommended searches, pin searches that you find most useful, and take advantage of organization-wide saved searches.
+Sentry allows you to save searches that you find most useful, either for yourself or for your entire organization.
```

Comment:
We want to tell people immediately that this only applies to the Issues page since this page isn't in the Issues section of our docs. Also avoiding passive voice in that second sentence. 

I'm also wondering if we need to introduce either here or in the intro of the custom section that the ability to make these org-wide is restricted to owners and managers. 

```suggestion
Sentry allows you to save searches you've made on the **Issues** page, either for yourself or for your entire organization. You can access these in the "Saved Searches" sidebar by clicking the "Saved Searches" button at the top right of the page.

```